src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs

```diff
@@ -168,6 +169,12 @@ public string TizenExpansionPackageName
         /// <since_tizen> 3 </since_tizen>
         public IEnumerable<string> Privileges { get { return _privileges; } }
 
+        /// <summary>
+        /// First installed time of the package.
+        /// </summary>
```

Comment:
It would be better if both InstalledTime and FirstInstalledTime have the same type, but I understand the reason behind current API.

Please consider adding additional information. My suggestion is based on the content of PR message.
```suggestion
        /// </summary>
        /// <remarks>
        /// This timestamp is not changed when application is updated, unlike the <see cref="InstalledTime"/>.
        /// When the application is uninstalled, this information is lost / not preserved.
        /// </remarks>
```